src/objecttypes/core/admin.py

```diff
@@ -139,3 +151,25 @@ def response_change(self, request, obj):
             return self.add_new_version(request, obj)
 
         return super().response_change(request, obj)
+
+    def import_github_url(self, request):
+        if request.method == "POST":
+            url = request.POST.get("github_url")
```

Comment:
Could you please add the form to validate user input:
* the string could not be a valid url
* the url could not exist
* it could not be json
* the json could not be a valid JSON schema

Now all these options result in 500 error

And could you also add unit tests which cover all these bullet points?